### ISSUE TITLE
Re-opening D-Tale Chart Colors GitHub Issue #325?

### ISSUE THREAD

**Issue by Valence-Shinei** [man-group/dtale#843]
I'm experiencing the same issue as described in 

[github.com/man-group/dtale/issues/325](github.com/man-group/dtale/issues/325)

😰.

I tried to understand the fix, but is there a configuration setting or file I need to change somewhere? I don't see a colorway option in the D-Tale interface and the demonstrated export issues shared in the above issue makes it difficult to see due to the blue on blue lack of contrast when exporting charts as .html. Also, the export .png seems to simply spin for 10+ min ...

My apologies if this isn't the proper way to engage with this issue 😅!

I'm currently running D-Tale 3.8.1 and Plotly 5.9.0.

Thank you in advance for your time and consideration of my request 🙏!

**Comment by aschonfeld** [comment]
@Valence-Shinei I see what you're talking about with the background being different in the PNG export versus the actual app:
![scatter_export_1706644494190](https://github.com/man-group/dtale/assets/11547371/8e66e468-b66c-4975-92ec-ac2565b1dcf0)
![image](https://github.com/man-group/dtale/assets/11547371/b5fe57e8-d13d-4186-ba97-bb61f8a84c4d)

I'll look into why its different.  There isn't a way to change the colorway from a user perspective.  It was just a small code update in that previous [issue](https://github.com/man-group/dtale/issues/325).

As for the spinning I can't really recreate that unless I just try to create a scatter with a ton of points, but then just trying to create the chart , let alone one that can be exported to a PNG, just spins forever.

**Comment by aschonfeld** [comment]
ok I've got it looking like this now:
![scatter_export_1706645516763](https://github.com/man-group/dtale/assets/11547371/1ad93c93-e589-4320-9c76-f06a709ba363)

I'll keep you posted on when this update is released

**Comment by Valence-Shinei** [comment]
Thank you kindly Andrew 🤝! That was precisely the concern I was noticing, especially since the white background provides a greater contrast against the blue plot points.

To clarify on the spinning 😅, I was noticing this issue when exporting to a self contained .html file, which had the same issue you demonstrated above. Such a .html file would generate quickly, less than 30 seconds, and would be about 7 MB in size. When I attempted to export instead as a .png file, I was unsuccessful, since the new tab in Chrome would simply spin. I think I killed the tab after 10 min. I tried this a couple of times without success.

However, since you were able to produce the above, it must be an issue with my machine that I'll have to troubleshoot. I prefer the .html file and its interactivity over the .png anyway 😉.

Thank you for clarifying on the colorway configuration as well. I appreciate you taking a look at this so quickly and I look forward to the update when it is ready (no rush 🤝)!
